apps/app/pages/index.tsx

```diff
@@ -79,8 +99,71 @@ function Dashboard() {
                 title="Tens a certeza que queres notificar?"
                 cancelText="Não"
                 okText="Sim"
-                onConfirm={(_) => notify_signup_ninjas()}
+                onConfirm={(_) => {
+                  notify_signup_ninjas();
+                  setConfirmed_signup(true);
+                }}
+                disabled={confirmed_signup}
               >
+                <Modal
+                  visible={visible_signup}
+                  onCancel={handleCloseModal_signup}
+                  centered={true}
+                  closable={false}
+                  title="Lista de Emails"
+                  footer={[
+                    <Button
+                      key="close"
+                      type="primary"
+                      onClick={handleCloseModal_signup}
+                    >
+                      Close
+                    </Button>,
+                  ]}
+                >
+                  <div
+                    style={{ display: "flex", justifyContent: "space-between" }}
+                  >
+                    <div
+                      style={{
+                        width: "45%",
+                        height: "500px",
+                        overflowY: "scroll",
+                        overflowX: "scroll",
+                      }}
```

Comment:
This specific style is being repeated in some parts of the code and that destructs the code reusability. A better way would be to have a `style.module.css`.